src/authorizations/components/redesigned/ResourceAccordionHeader.tsx

```diff
@@ -65,9 +65,6 @@ export const ResourceAccordionHeader: FC<OwnProps> = props => {
         stretchToFitWidth={true}
         alignItems={AlignItems.Center}
         style={{textAlign: 'start'}}
-        onClick={(e: any) => {
```

Comment:
The `FlexBox` does not have `onClick` anymore, does this sound right @daviddkkim?

Reply:
The onclick prop addition to flexbox happened to clockface 2.0 after we created clockface 3.0. So 3.0 will have it as well!

Reply:
I merged the newest clockface updates to the beta version and published a new version in `3.0.0-beta.3`

Reply:
Got you! So I will add this back after pulling in the latest `clockface`